libdnf/conf/ConfigMain.cpp

```diff
@@ -348,6 +348,7 @@ class ConfigMain::Impl {
     OptionBool deltarpm{true};
     OptionNumber<std::uint32_t> deltarpm_percentage{75};
     OptionBool skip_if_unavailable{false};
+    OptionBool sslverify_status{false};
```

Comment:
Just passing by, really...

Why is the option named `sslverify_status` here? The librepo option is called `LRO_SSLVERIFYSTATUS` (no underscores besides the `LRO_` prefix), the curl option is called `CURLOPT_SSL_VERIFYSTATUS` (an underscore between SSL and VERIFYSTATUS), from the librepo PR: https://github.com/rpm-software-management/librepo/pull/213 So we have multiple variants of underscore placement... :slightly_smiling_face: 

Is this different from the `sslVerify` option e.g. git has?

Reply:
For the liberepo names I tried to keep it in line with existing:
- `LRO_SSLVERIFYPEER` (curl: `CURLOPT_SSL_VERIFYPEER`)
- `LRO_SSLVERIFYHOST` (curl: `CURLOPT_SSL_VERIFYHOST`)

I think the git option `sslVerify` is more in line with dnf option `sslverify` but I havent't check if they do exactly the same stuff. For dnf it actually sets the librepo's `LRO_SSLVERIFYPEER` and `LRO_SSLVERIFYHOST`.

The option I am adding here is specifically for certificate revocation checking using [OCSP stapling](https://curl.se/libcurl/c/CURLOPT_SSL_VERIFYSTATUS.html). It needs a separate option because it is not supported that much. More analogous git option for that would be `http.schannelCheckRevoke` but they have it specific to schannel backend it seems, which may not even use OCSP.

However I am not sure if `sslverify_status` is a good name, I tried to keep the 'status' from the curl option, but maybe something with revocation checking and/or OCSP would be better.

Reply:
I'd say keep the "status" in the name, it kinda does describe it's checking the certificate status, though to an unknowing user it may sound generic and may be confused with e.g. the verifypeer or verifyhost.

No need to get stuck bikeshedding the name, I see the other ssl options in libdnf conf have no underscores (which is unfortunate), so I'd either (preferably?) go with that and use `sslverifystatus` or set a new direction with the naming and use `ssl_verify_status`? But feel free to keep the one you have now if you actually prefer that.

Reply:
Yes I think `sslverifystatus` is the best approach here. Thanks!
(I still need to update the docs and I am waiting to see if its actually acceptable solution for the bz, but it looks promising)